cogs/tracking.py

```diff
@@ -1282,11 +1280,18 @@ def __init__(self, bot: NabBot):
                     continue
                 await conn.execute("INSERT INTO character_death_killer(death_id, name, player) VALUES($1, $2, $3)",
                                    death_id, death.killer, death.by_player)
-                if time.time() - death.time.timestamp() >= (30 * 60):
-                    log.info(f"Death detected, too old to announce: {char.name}({death.level}) | {death.killer}")
+                log_msg = f"Death detected: {char.name}({death.level}) | {death.killer}"
+                if self.is_old_death(death):
+                    log_msg += ", but it is too old to announce."
                 else:
+                    log.info(log_msg)
                     await self.announce_death(char, death, max(death.level - char.level, 0))
 
+    @staticmethod
+    def is_old_death(death):
```

Comment:
Why add a method for something that is one line and is only used in one place? 🤔

Reply:
Readability/easier to understand.

Reply:
I wrote the comment because to me (inexperienced with python) it is not clear upfront what the `30 * 60` is achieving. I'm used to count time in milliseconds, those were seconds to minutes. 

I think that bigger and more complex methods should be extracted into smaller methods so we can also give them meaningful names, separate their purposes and also facilitate readability.

You say the comment will help, but in theory the comment should not be needed.
Maybe it would make more sense if I refactored it better, in this case I only refactored one small part of code.

For instance, I think the method should look somewhat like this (obviously summarizing):
```
async def compare_deaths(self, char: Character):
    """Checks if the player has new deaths, then save and announce them."""
    if char:
        async with self.bot.pool.acquire() as conn:
            char_id = get_char_from_db(char, conn)
            if char_id:
	        pending_deaths = get_pending_deaths(char_id)
	        save_new_deaths(pending_deaths)
                announce_new_deaths(pending_deaths)
``` 

I should get a good idea of what the whole method is doing by my first glances at it. It is easier to do that if it has smaller parts with better naming.